netd.go

```diff
@@ -17,17 +17,17 @@ package main
 
 import (
 	"flag"
-	"github.com/samalba/dockerclient"
-	"log"
-	"os"
-	"strings"
-
 	"github.com/contiv/netplugin/core"
 	"github.com/contiv/netplugin/crt"
 	"github.com/contiv/netplugin/crtclient"
 	"github.com/contiv/netplugin/crtclient/docker"
 	"github.com/contiv/netplugin/drivers"
 	"github.com/contiv/netplugin/plugin"
+	"github.com/samalba/dockerclient"
+	"log"
```

Comment:
The usual convention I have seen is that the standar packages go at the top and then the other user packages with a empty line between the two. We should follow that here for consistency.